whylabs_toolkit/helpers/utils.py

```diff
@@ -1,12 +1,29 @@
+from typing import Union
+
 from whylabs_client.api.dataset_profile_api import DatasetProfileApi
 from whylabs_client.api.models_api import ModelsApi
+from whylabs_client.api.notification_settings_api import NotificationSettingsApi
 
-from whylabs_toolkit.helpers.client import client
+from whylabs_toolkit.helpers.client import create_client
+from whylabs_toolkit.monitor.models import SlackWebhook, EmailRecipient, GlobalAction
 
 
 def get_models_api() -> ModelsApi:
-    return ModelsApi(api_client=client)
+    return ModelsApi(api_client=create_client())
 
 
 def get_dataset_profile_api() -> DatasetProfileApi:
-    return DatasetProfileApi(api_client=client)
+    return DatasetProfileApi(api_client=create_client())
+
+
+def get_notification_api() -> NotificationSettingsApi:
+    return NotificationSettingsApi(api_client=create_client())
+
+
+def get_notification_request_payload(action: Union[SlackWebhook, EmailRecipient]) -> str:
+    if isinstance(action, SlackWebhook):
+        return "slackWebhook"
+    elif isinstance(action, EmailRecipient):
+        return "email"
+    else:
+        raise ValueError(f"Can't work with {action} type")
```

Comment:
Is this something the user can fix on their own when it happens? If not then we should say that its a bug in the library so they have some hint of what to do when this happens. Same applies to other exceptions.

Reply:
what would they fix? this is just a helper method to be able to use `PUT` correctly for each action. thing is that they are only allowed to auto-put with email or slack. other stuff they will need to 1. create it first and 2. use global actions. 
maybe the best is that this is just a private static method built inside of the Monitor Manager?